### ISSUE TITLE
Is modality feature extractor available? 

### ISSUE THREAD

**Issue by MrZilinXiao** [hujingwen6666/MMGCN#2]
Thank you for your great work! 
And may I ask if you could provide scripts of the feature extractor that produce `.pkl` files in your repo? Thank you in advance.

**Comment by BladeDancer957** [comment]
> 

Do you find scripts of the feature extractor?  I need them too.

**Comment by MrZilinXiao** [comment]
> > 
> 
> 
> 
> Do you find scripts of the feature extractor?  I need them too. 

Nope. We may contact authors via email.😊

**Comment by MatheusMMatos** [comment]
@MrZilinXiao did you got the scripts? or something similar?

**Comment by MrZilinXiao** [comment]
> @MrZilinXiao did you got the scripts? or something similar?

Nope. If you are unable to seek alternative, you might try to just use the features the author provided with.

**Comment by Coding511** [comment]
@MrZilinXiao did you run the code with the given features?
I am getting the error:
**NotImplementedErro**r

at line 360

While executing train.py

The
 path = './saved/IEMOCAP/'

is not there,

**Comment by purpleofdial** [comment]
@MrZilinXiao May I ask whether the feature given by the author is the raw feature written in the article?
